### ISSUE TITLE
What about OVH ?

### ISSUE THREAD

**Issue by AVeylon** [rmbolger/Posh-ACME#79]
Hi, i'm trying to use a DNS Plugin for OVH but I don't manage to find one.

Is it a plugin already existing for OVH but with an other name ?

Thank you !

**Comment by rmbolger** [comment]
Nope. I can take a look at them and see if I can whip one up though.  Just to clarify, you're talking about [this OVH](https://www.ovh.com/world/)?

**Comment by AVeylon** [comment]
Yes this one absolutely.
Would be really cool !

Thanks

**Comment by rmbolger** [comment]
So I tried creating an account. But I can't seem to find where the DNS services are in the Control Panel. Are they in the Public Cloud section? If so, it looks like they want me to pre-pay like $40 to activate the Public Cloud feature which I can't really justify out of pocket.

Let me see if I can find another way to do this.

**Comment by AVeylon** [comment]
I think you can't find the DNS Zone until you created a domain name in OVH.
It looks like you won't be able to do this ... thanks for the answer tho, do you have any advice so I can try creating it myself ?
Thanks

**Comment by Studisys** [comment]
Hello,

I don't know I you're aware of it, but there is an API for OVH, that is publicly accessible (even without any account) : https://api.ovh.com/console/
You may find useful things in it ;-)

**Comment by rmbolger** [comment]
The links @Studisys are a great form of documentation and should definitely help if you want to try writing the plugin yourself. You should also review the [Plugin Development Guide](https://github.com/rmbolger/Posh-ACME/blob/master/Posh-ACME/DnsPlugins/README.md). I'm happy to help if you get stuck.

**Comment by rmbolger** [comment]
It seems my original problem is that not all OVH regions appear to support DNS hosting. So I had created an account in the US region, but there's no support for adding/creating a new domain there. Once I figured out I could create an account in the EU region, things worked much better.

The plugin hypothetically supports all regions from an API standpoint. But obviously that's going to be dependent on whether your particular region actually supports hosting DNS. And I only tested it against OVH EU. The [usage guide](https://github.com/rmbolger/Posh-ACME/blob/master/Posh-ACME/DnsPlugins/OVH-Readme.md) has more details.
